src/CMakeLists.txt

```diff
@@ -785,6 +785,7 @@ add_definitions(-D__HEAP_SIZE=4096)
 
 # Target hardware configuration options
 add_definitions(-DTARGET_DEVICE_${TARGET_DEVICE})
+add_definitions(-DTARGET_DEVICE_NAME="${TARGET_DEVICE}")
 if(TARGET_DEVICE STREQUAL "PINETIME")
```

Comment:
Non blocking. I'm wondering if the model should not be all caps if it will be visible, so `PineTime`

Reply:
Fair point. Since the name is derived from the build configuration, there would have to be some kind of lookup / conditional in the CMake config. That would not be a problem, we could even prefix the `MOY` variants with e.g. `P8`. 

On the other hand, this info is purely for informational purposes (it is on page two of the system information ..), so aesthetic requirements are somewhat negotiable imho.

I am neutral on this, what do you think?

Reply:
I guess it's best practices to keep the definition all caps(, or all lowercase?). I don't think it's worth adding more code to get around this.

Reply:
I agree that consistency is desirable. CMake enforces all-caps in https://github.com/InfiniTimeOrg/InfiniTime/blob/develop/CMakeLists.txt#L58= , so we might as well keep it that way.